E2E-Demos/Investment-Banking/Airflow/spark_etl_new.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: "sparkoperator.hpe.com/v1beta2"
+kind: SparkApplication
+metadata:
+  name: "spark-etl"
+  namespace: "spark"
+spec:
+  sparkConf:
+    # S3 credentials should be specified below
+    spark.hadoop.fs.s3a.endpoint: "https://home.hpe-apps-ezaf.com:31900"
```

Comment:
can we avoid hardcoding domain name, but use internal fqdn for s3, i.e., 'https://objectstore-zone1-svc.dataplatform.svc.cluster.local:9000/'